gaffer-core/graph/src/main/java/gaffer/graph/Graph.java

```diff
@@ -107,7 +116,17 @@ private Graph(final Store store, final View view) {
             }
         }
 
-        return store.execute(operationChain, user);
+        for (GraphHook graphHook : graphHooks) {
```

Comment:
gh-220 - Add final to enhanced loop variables to ensure that they aren't updated inside of the loop